src/containers/datasets/biomass/hooks.tsx

```diff
@@ -25,14 +32,33 @@ const getColorKeys = (data: Data[]) =>
   }, {} satisfies ColorKeysTypes);
 // widget data
 export function useMangroveBiomass(
-  params: UseParamsOptions,
-  queryOptions: UseQueryOptions<DataResponse>
+  params?: UseParamsOptions,
+  queryOptions?: UseQueryOptions<DataResponse>
 ): BiomassData {
+  const currentYear = useRecoilValue(BiomassYearSettings);
+  const {
+    query: { locationType, id },
+  } = useRouter();
```

Comment:
This won't work after my changes in the routing, please access to `query.params` array from now on. https://github.com/Vizzuality/mangrove-atlas/blob/0ce8591c207d760f3cf4aca78222ff42ea31c60e/src/components/location-title/index.tsx#L13-L17